### PR TITLE
Support image as numpy array

### PR DESCRIPTION
**Pull request by masadcv**
Let me start by saying many thanks for a really nice library.

We are working with numpy arrays that have been loaded and oriented using nibabel library. Therefore, I did a quick fix to make this work with numpy arrays that are already in correct orientation. If you think this is useful, would be good to merge with master. 

Let me know your thoughts/comments on this. Thanks!

### CODE REVIEW

File: lungmask/mask.py [hunk 1]
```diff
@@ -23,11 +23,15 @@
 def apply(image, model=None, force_cpu=False, batch_size=20, volume_postprocessing=True, noHU=False):
     if model is None:
         model = get_model('unet', 'R231')
-
-    inimg_raw = sitk.GetArrayFromImage(image)
-    directions = np.asarray(image.GetDirection())
-    if len(directions) == 9:
-        inimg_raw = np.flip(inimg_raw, np.where(directions[[0,4,8]][::-1]<0)[0])
+    
+    numpy_mode=isinstance(image, np.ndarray)
```
Comment (JoHof) [inline lungmask/mask.py:27]: can you please add spaces between the `=` for a consistent code style?

**Review by JoHof — CHANGES_REQUESTED** [review]
Hi, thanks the PR. Could you also increase the version number to 0.2.9 and add a section in the readme describing the axis when providing a numpy array (first axis -> slices, second  axis -> chest to back, third axis -> right to left). Thanks!

File: lungmask/mask.py (hunk 1 above)
Comment (masadcv) [reply lungmask/mask.py:27]: Many thanks for the review! I will update with changes shortly.

**Comment by masadcv** [comment]
All addressed, let me know if anything else is needed for this.

**Review by JoHof — APPROVED** [review]
LGTM, thanks!

**Pull request merged**